source/AlarmWindow.cpp

```diff
@@ -53,24 +56,24 @@ AlarmWindow :: AlarmWindow (BRect frame, BHandler *handler)
     char 	hourDefaultField[3];
 
 	// Initialize text fields with current system time values
-	sprintf(minuteDefaultField, "%d", GetTime(0));
-	sprintf(hourDefaultField, "%d", GetTime(1));
-	sprintf(dayDefaultField, "%d", GetTime(2));
-	sprintf(monthDefaultField, "%d", GetTime(3));
+	sprintf(minuteDefaultField, "%02d", GetTime(0));
+	sprintf(hourDefaultField, "%02d", GetTime(1));
+	sprintf(dayDefaultField, "%02d", GetTime(2));
+	sprintf(monthDefaultField, "%02d", GetTime(3));
 	sprintf(yearDefaultField, "%d", GetTime(4));
 
 	// We allocate the view AlarmView and associate it to the AlarmWindow
 	frame.OffsetTo(B_ORIGIN);
 	fAlarmView = new AlarmView(frame,"AlarmView");
 	fAlarmView->SetViewColor(ui_color(B_PANEL_BACKGROUND_COLOR));
-	AddChild(fAlarmView);
+	//AddChild(fAlarmView);
```

Comment:
Remove commented line.
All widgets should use the layout aware versions, the ones without the BRect parameter.